lib/geokit/geocoders.rb

```diff
@@ -1,14 +1,13 @@
-require 'net/http'
-require 'openssl'
 require 'geokit/net_adapter/net_http'
 require 'geokit/net_adapter/typhoeus'
 require 'ipaddr'
+require 'json'
+require 'logger'
+require 'net/http'
+require 'openssl'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.